README.adoc

```diff
@@ -234,7 +235,8 @@ The following Maven plugin is responsible for downloading and installing the fea
 </plugin>
 ----
 
-This user feature also comes with the `opentracing-1.0` feature, so you don't need to enable it explicitly.
+This user feature also comes with the `opentracing-1.0` feature, which it requires and which will be
+enabled automatically.
```

Comment:
I don't think the user feature comes with `opentracing-1.0` but `opentracing-1.0` is there in OL and the user feature simply enables/requires it.  I'd simply drop the mention of `opentracing-1.0` if it doesn't add to the guide as our focus here is on mpOpenTracing-1.0.

Reply:
Ok sure